addons/website/static/src/snippets/s_table_of_content/000.js

```diff
@@ -11,18 +12,14 @@ const TableOfContent = publicWidget.Widget.extend({
      * @override
      */
     start: function () {
-        const $menuLoading = $('.o_menu_loading');
-        if (!$menuLoading.length) {
-            this._initializeNavbarTopPosition();
-        } else {
-            $(document.body).one('menu_loaded', () => this._initializeNavbarTopPosition());
-        }
```

Comment:
Why removing this?

Reply:
now _initializeNavbarTopPosition is only usefull if header effect is "scrolled". So we dont need to wait menu loading because the menu is not a fixed element